common/src/com/intellij/plugins/haxe/config/HaxeTarget.java

```diff
@@ -33,6 +33,7 @@
   // Target     clFlag    Extension,  OutputDir,  OutputType              Description
 
   NEKO(         "neko",   ".n",       "neko",     OUTPUT_TYPE.FILE,       HaxeCommonBundle.message("haxe.target.neko")),
+  HASHLINK(     "hl",     ".hl",      "hl",       OUTPUT_TYPE.FILE,       HaxeCommonBundle.message("haxe.target.hl")),
```

Comment:
Why not just use the HL already defined at line 45(46)?  And, if it's not used anywhere else, we should probably just have one.

Reply:
a bit  tired i guess, fixed now